packages/flame_isolate/README.md

```diff
@@ -0,0 +1,46 @@
+The power of [integral_isolates](https://pub.dev/packages/integral_isolates) in your
```

Comment:
Maybe start the readme with a simple `# Flame Isolate` or `# flame_isolate` (and then add a # to the other sections.